src/button.test.aria.ts

```diff
@@ -0,0 +1,26 @@
+import { expect, test } from '@playwright/test';
+import type GlideCoreButton from './button.js';
+
+test.describe('button--button', () => {
```

Comment:
We should have a lot fewer ARIA tests than we do visual ones because there are just far fewer states the accessibility tree can be in. We don't, for example, need to test every `variant` because changes to `variant` don't affect the accessibility tree.